tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -574,10 +574,21 @@ module.exports = {
      */
     getExpirationDateFromInputField: async function () {
       let expirationDate
-      await this.waitForElementVisible('@expirationDateField').getValue(
-        '@expirationDateField',
+      await this.waitForElementVisible('xpath', '//*[@data-testid="recipient-datepicker"]')
```

Comment:
Please define a new element (see bottom section of this file) and use that here. Elements (e.g. `xyz`) can be referenced by variable (`this.elements.xyz`) or via string (`@xyz`). Gives the file more structure and makes refactoring and reuse easier.

Reply:
Afaik the string reference (`@xyz`) automatically resolves the `use` parameter, so you can omit the `xpath` param in both lines and just give it the string ref.